CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### Features
+
+### Bug Fixes
+
+### Other
+
```

Comment:
I don't think these lines should be here? The file should just start with `## v1.10.2 [2023-10-20]` and then the changes relevant to that version

Reply:
fixed.
I've followed https://github.com/influxdata/chronograf_/blob/master/RELEASE.md but probably midunderstood some it, sorry.